src/applications/disability-benefits/526EZ/sass/disability-benefits.scss

```diff
@@ -138,14 +138,10 @@
   }
 }
 
-.disability-increase-wizard {
-
-  .fieldset-input {
-
-    .legend-label {
-      max-width: 100%;
-    }
-
-  }
+.disability-increase-wizard .fieldset-input .legend-label {
+  max-width: 100%;
+}
 
+.fetch-fail-alert {
```

Comment:
Should we use the component name here .async-display-widget or something, as well?

Reply:
Yeah, good idea. :+1: